components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import type { AccordionItemType } from '@/types/components/AccordionItemType';
+
+import AccordionItem from './AccordionItem';
+
+export interface AccordionProps {
+  accordionItems: AccordionItemType[];
+}
+
+const Accordion = ({ accordionItems = [] }: AccordionProps) => {
```

Comment:
use `export default function` here and create a JSdoc above this function to describe the component.

Reply:
Done